ipset/ipset.go

```diff
@@ -25,8 +25,8 @@ import (
 	"strconv"
 	"strings"
 
-	log "github.com/Sirupsen/logrus"
 	"github.com/coreos/go-semver/semver"
+	"github.com/sirupsen/logrus"
```

Comment:
Keep the log package identifier. 
It makes it easier to switch for another implementation later on if required.